### PR TITLE
feat(docs): gcutil is deprecated; convert all calls to gcloud compute

### PR DESCRIPTION
**Pull request by wenzowski**
continuation of #3123

updates to recommended defaults per that thread should probably be reviewed once more before merge

### CODE REVIEW

File: docs/installing_deis/gce.rst [hunk 1]
```diff
@@ -95,47 +95,41 @@ Create a SSH key that we will use for Deis host communication:
...
+
+    NAME  ZONE          MACHINE_TYPE INTERNAL_IP   EXTERNAL_IP    STATUS
+    core1 us-central1-f n1-highmem-2 10.240.10.107 108.59.80.10   RUNNING
+    core2 us-central1-f n1-highmem-2 10.240.10.108 108.59.80.11   RUNNING
+    core3 us-central1-f n1-highmem-2 10.240.10.109 108.59.80.12   RUNNING
 
 .. note::
 
-    The provision script will by default provision ``n1-standard-2`` instances. Choosing a smaller
+    The provision script will by default provision ``n1-highmem-2`` instances. Choosing a smaller
```
Comment (carmstrong) [inline docs/installing_deis/gce.rst]: What's the motivation for this change? Is `standard` not large enough? If we want to make this change, it should be at the minimum a separate commit (or even a separate PR).

**Comment by carmstrong** [comment]
Thanks for keeping us up-to-date, @wenzowski. :+1:

File: docs/installing_deis/gce.rst (hunk 1 above)
Comment (wenzowski) [inline docs/installing_deis/gce.rst]: The failure rate dropped dramatically. Can definitely split this out.

**Comment by wenzowski** [comment]
my pleasure @carmstrong

**Comment by wenzowski** [comment]
running a test cluster with 64GB persistent ssd now. At UX level, difference between ephemeral direct-attached ssd & 64GB persistent ssd pool not obvious. And much safer. :wink:

**Comment by wenzowski** [comment]
Test cluster with this config has been working smoothly now for 2 days; a far cry from the original defaults. I'm using n1-highmem-2 instances, but aside from that config is as documented.

While this is zone f, the default should probably be [availability zone c](https://cloud.google.com/compute/docs/zones#available) right now. GCE Haswell instances cost no more than Sandy Bridge or Ivy Bridge instances.

File: docs/installing_deis/gce.rst [hunk 2]
```diff
@@ -95,43 +95,37 @@ Create a SSH key that we will use for Deis host communication:
...
-    +--------+---------------+--------+---------+
-    | name   | zone          | status | size-gb |
-    +--------+---------------+--------+---------+
-    | cored1 | us-central1-a | READY  |      32 |
-    +--------+---------------+--------+---------+
-    | cored2 | us-central1-a | READY  |      32 |
-    +--------+---------------+--------+---------+
-    | cored3 | us-central1-a | READY  |      32 |
-    +--------+---------------+--------+---------+
+    $ gcloud compute disks create cored1 cored2 cored3 --size 64GB --type pd-ssd --zone us-central1-f
```
Comment (mboersma) [inline docs/installing_deis/gce.rst]:
```console
$ gcloud compute disks create cored1 cored2 cored3 --size 64GB --type pd-ssd --zone us-central1-f
NAME ZONE SIZE_GB TYPE STATUS
ERROR: (gcloud.compute.disks.create) Some requests did not succeed:
 - Invalid value for field 'resource.diskTypes': 'projects/testing-deis-001/zones/us-central1-f/diskTypes/pd-ssd'.  Resource was not found.
$ gcloud compute disk-types list
NAME        ZONE           VALID_DISK_SIZES
pd-standard europe-west1-d 10GB-10240GB
pd-standard asia-east1-a   10GB-10240GB
pd-standard asia-east1-b   10GB-10240GB
pd-standard asia-east1-c   10GB-10240GB
pd-standard europe-west1-a 10GB-10240GB
pd-standard europe-west1-b 10GB-10240GB
pd-standard europe-west1-c 10GB-10240GB
pd-standard us-central1-a  10GB-10240GB
pd-standard us-central1-c  10GB-10240GB
pd-standard us-central1-b  10GB-10240GB
pd-standard us-central1-f  10GB-10240GB
```
Should this be `pd-standard`? That's all I see available.

**Comment by mboersma** [comment]
Tested on GCE, all commands worked fine with copy-and-paste except I had to omit the `--type pd-ssd` parameter as noted above (defaulting to `pd-standard`).

**Comment by wenzowski** [comment]
I found ceph unreliable with the pd-standard which is why the move to pd-ssd from #3123

File: docs/installing_deis/gce.rst (hunk 2 above)
Comment (wenzowski) [inline docs/installing_deis/gce.rst]: I'm not sure what's going on there
```
gcloud compute disk-types list
NAME        ZONE           VALID_DISK_SIZES
pd-ssd      europe-west1-d 10GB-10240GB
pd-standard europe-west1-d 10GB-10240GB
local-ssd   asia-east1-a   375GB-375GB
pd-ssd      asia-east1-a   10GB-10240GB
pd-standard asia-east1-a   10GB-10240GB
local-ssd   asia-east1-b   375GB-375GB
pd-ssd      asia-east1-b   10GB-10240GB
pd-standard asia-east1-b   10GB-10240GB
local-ssd   asia-east1-c   375GB-375GB
pd-ssd      asia-east1-c   10GB-10240GB
pd-standard asia-east1-c   10GB-10240GB
pd-ssd      europe-west1-a 10GB-10240GB
pd-standard europe-west1-a 10GB-10240GB
local-ssd   europe-west1-b 375GB-375GB
pd-ssd      europe-west1-b 10GB-10240GB
pd-standard europe-west1-b 10GB-10240GB
local-ssd   europe-west1-c 375GB-375GB
pd-ssd      europe-west1-c 10GB-10240GB
pd-standard europe-west1-c 10GB-10240GB
local-ssd   us-central1-a  375GB-375GB
pd-ssd      us-central1-a  10GB-10240GB
pd-standard us-central1-a  10GB-10240GB
local-ssd   us-central1-c  375GB-375GB
pd-ssd      us-central1-c  10GB-10240GB
pd-standard us-central1-c  10GB-10240GB
pd-ssd      us-central1-b  10GB-10240GB
pd-standard us-central1-b  10GB-10240GB
local-ssd   us-central1-f  375GB-375GB
pd-ssd      us-central1-f  10GB-10240GB
pd-standard us-central1-f  10GB-10240GB
```

Comment (wenzowski) [inline docs/installing_deis/gce.rst]: old gcloud version?
```
gcloud --version
Google Cloud SDK 0.9.50

bq 2.0.18
bq-nix 2.0.18
compute 2015.03.03
core 2015.03.03
core-nix 2014.10.20
dns 2015.03.03
gcutil 1.16.5
gcutil-nix 1.16.5
gsutil 4.9
gsutil-nix 4.6
sql 2015.03.03
```

Comment (wenzowski) [inline docs/installing_deis/gce.rst]: If yours are different @mboersma would you mind doing a
```
gcloud components update
```
and try again?

Comment (mboersma) [inline docs/installing_deis/gce.rst]: No, it auto-updated just before I ran `gcloud`:
```console
$ gcloud --version
Google Cloud SDK 0.9.51

bq 2.0.18
bq-nix 2.0.18
compute 2015.03.10
core 2015.03.10
core-nix 2014.10.20
dns 2015.03.10
gcutil 1.16.5
gcutil-nix 1.16.5
gsutil 4.10
gsutil-nix 4.6
sql 2015.03.10
```

Comment (mboersma) [inline docs/installing_deis/gce.rst]: I'm on a GCE trial account. Maybe they limit my options?

Comment (wenzowski) [inline docs/installing_deis/gce.rst]: ah, yep that'll be it

Comment (wenzowski) [inline docs/installing_deis/gce.rst]: hmm, that complicates things

guess the default should stay `--pd-standard` with a note that `--pd-ssd` is recommended?

Comment (mboersma) [inline docs/installing_deis/gce.rst]: That's probably the safest choice, or making `--type` an environment variable or arg to the script.

Comment (wenzowski) [inline docs/installing_deis/gce.rst]: I think a GCE spinup script should probably be a separate PR.

Comment (wenzowski) [inline docs/installing_deis/gce.rst]: --pd-ssd pools are capped at a pitiful 1TB per zone by default, so folks should probably just be using the direct-attached ssds
